src/upsell_panda.js

```diff
@@ -127,11 +125,8 @@ function showNextPromo() {
   }
 
   var nextPromotion = promotionsList[iterator];
-  renderPromotionInContainer(nextPromotion);
-
-  if (nextPromotion.hasCountdown) {
-    turnOnCountdown(nextPromotion);
-  }
+  var fadeIn = promotionsList.length > 1;
```

Comment:
We only want fade in animation if there is more than one promotion.

Reply:
Okay, good call.